library/panic_unwind/src/emcc.rs

```diff
@@ -89,7 +88,7 @@ pub unsafe fn panic(data: Box<dyn Any + Send>) -> u32 {
     let sz = mem::size_of_val(&data);
     let exception = __cxa_allocate_exception(sz) as *mut Exception;
     if exception.is_null() {
-        return uw::_URC_FATAL_PHASE1_ERROR as u32;
+        return 0;
```

Comment:
This change feels unrelated to the stated goal in the PR description?

Reply:
Yes it's unrelated, I just noticed it while going through the code. Emscripten doesn't use the libunwind API and shouldn't be returning libunwind error codes. In the end it doesn't matter since the returned value is not used for anything, it's just printed as part of the message that indicates that initiating unwinding failed.